recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+export DYLD_LIBRARY_PATH=$PREFIX/lib
+
+# https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg11423.html
+export LD_LIBRARY_PATH=$PREFIX/lib
+
+./configure \
```

Comment:
So, I know @groutr [mentioned]( https://github.com/conda-forge/staged-recipes/pull/192#discussion_r57233201 ) that he tried `cmake` to build `hdf5` and had issues. The issues may also present themselves here so we should be careful. Personally, given `hdf5` had issues, I prefer `configure` at present.